src/CMakeLists.txt

```diff
@@ -344,20 +376,11 @@ install(DIRECTORY ${PROJECT_SOURCE_DIR} DESTINATION include
 	  FILES_MATCHING PATTERN "nupic/*/*.h*"
 	  PATTERN "nupic/*/*.hpp.in" EXCLUDE)
 
-install(DIRECTORY ${REPOSITORY_DIR}/external/common/include/gtest
-	  MESSAGE_NEVER
-	  DESTINATION include/gtest
-	  FILES_MATCHING PATTERN "*.h*")
```

Comment:
don't we need gtest includes?

Reply:
> don't we need gtest includes?

We don't need it but I guess if the lib user wanted to use it to make their own unit tests for their own code it might be useful.  I don't know...should we provide it?

Reply:
no need. If ever such request appears, we can add it. 
So keep this removed.